### PR TITLE
chore: v1.9.1 Changelog

### PR DESCRIPTION
**Pull request by another-rex**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
+OSV-Scanner v2 is coming soon! The next release will start with version `v2.0.0-alpha1`.
+
+Here's a peek at some of the exciting upcoming features:
+
+- Standalone container image scanning support.
+  - Including support for Alpine and Debian images.
```
Comment (cuixq) [inline CHANGELOG.md:6]: remove space?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+- Refactored internals to use `osv-scalibr` library for better extraction capabilities.
```
Comment (cuixq) [inline CHANGELOG.md:7]: link to the `osv-scalibr` repository?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+This is the final feature v1 release of osv-scanner, future releases for v1 will only contain bug fixes.
+
+# v1.9.1
+
+### Features:
+
+- [Feature #1295](https://github.com/google/osv-scanner/pull/1295) Support offline database in fix subcommand.
+- [Feature #1342](https://github.com/google/osv-scanner/pull/1342) Add `--experimental-offline-vulnerabilities` and `--experimental-no-resolve` flags.
+- [Feature #1045](https://github.com/google/osv-scanner/pull/1045) Support private registries for Maven.
+- [Feature #1226](https://github.com/google/osv-scanner/pull/1045) Support support `vulnerabilities.ignore` in package overrides.
```
Comment (cuixq) [inline CHANGELOG.md:26]: wrong URL

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,49 @@
...
+
+### Fixes:
+
+- [Bug #604](https://github.com/google/osv-scanner/pull/604) Use correct path separator in SARIF output when on Windows.
+- [Bug #330](https://github.com/google/osv-scanner/pull/330) Warn about and ignore duplicate entries in SBOMs.
+- [Bug #1325](https://github.com/google/osv-scanner/pull/1325) Set CharsetReader and Entity when reading pom.xml.
+- [Bug #1310](https://github.com/google/osv-scanner/pull/1310) Update spdx license ids.
+- [Bug #1288](https://github.com/google/osv-scanner/pull/1288) Sort sbom packages by PURL.
+- [Bug #1288](https://github.com/google/osv-scanner/pull/1288) Improve handling if `docker` exits with a non-zero code when trying to scan images
+-
```
Comment (cuixq) [inline CHANGELOG.md:36]: remove

File: CHANGELOG.md (hunk 3 above)
Comment (another-rex) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (another-rex) [reply CHANGELOG.md]: Done, though it looks like we also should update the osv-scalibr readme.

File: CHANGELOG.md (hunk 1 above)
Comment (another-rex) [reply CHANGELOG.md:6]: Do you mean the indent? That is intentional

**Review by oliverchang — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (cuixq) [reply CHANGELOG.md:6]: yes

**Review by cuixq — APPROVED** [review]

**Pull request merged**